ddtrace/contrib/falcon/middleware.py

```diff
@@ -3,29 +3,34 @@
 from ddtrace import config
 from ddtrace.ext import SpanTypes
 from ddtrace.ext import http as httpx
-from ddtrace.propagation.http import HTTPPropagator
 
 from .. import trace_utils
 from ...compat import iteritems
 from ...constants import ANALYTICS_SAMPLE_RATE_KEY
 from ...constants import SPAN_MEASURED_KEY
+from ...utils.formats import asbool
 
 
 class TraceMiddleware(object):
-    def __init__(self, tracer, service="falcon", distributed_tracing=True):
+    def __init__(self, tracer, service="falcon", distributed_tracing=None):
         # store tracing references
         self.tracer = tracer
         self.service = service
-        self._distributed_tracing = distributed_tracing
+        if distributed_tracing is not None:
+            self._distributed_tracing = distributed_tracing
+
+    @property
+    def _distributed_tracing(self):
```

Comment:
Probably don't need the property anymore since it's internal and it's never set or read again